frontend-react/src/utils/misc.ts

```diff
@@ -46,6 +46,38 @@ export const checkJson = (
     }
 };
 
+export function includesSpecialChars(text: string): boolean {
```

Comment:
Is it worth it to combine this with `isProhibitedOrgName`?  It's a pretty specific regular expression, and the name of the function is a little generic 🤔

Reply:
Good point since the same rules apply for a new org name in theory.

And thinking about it, maybe also Value Sets?

Reply:
I can combine it but had it as its own function if we were to use it elsewhere and also for testing purposes. I felt that it was unique enough to separate.

Reply:
Sure, not a big deal in my mind at this point.  I do think it's worth updating the name of the function to be a little more indicative of what it's checking for, but also not a blocker!